ArduCopter/mode.cpp

```diff
@@ -405,6 +405,7 @@ bool Copter::set_mode(const uint8_t new_mode, const ModeReason reason)
 // called at 100hz or more
 void Copter::update_flight_mode()
 {
+    pos_control->update_estimates(vibration_check.high_vibes);
```

Comment:
It would make me slightly happier if this was exactly where the old inav update was, just so its clear there is no change. We could then move it in a future PR if we need to.

Reply:
I have done this.